examples/multiple_tables/main.tf

```diff
@@ -23,9 +23,114 @@ module "bigquery" {
   default_table_expiration_ms = var.default_table_expiration_ms
   project_id                  = var.project_id
   location                    = "US"
-  tables                      = var.tables
-  dataset_labels              = var.dataset_labels
-  encryption_key              = var.kms_key
+  tables = [
+    {
+      table_id = "foo",
+      schema   = "sample_bq_schema.json",
+      time_partitioning = {
+        type                     = "DAY",
+        field                    = null,
+        require_partition_filter = false,
+        expiration_ms            = null,
+      },
+      expiration_time = null,
+      clustering      = ["fullVisitorId", "visitId"],
+      labels = {
+        env      = "dev"
+        billable = "true"
+        owner    = "joedoe"
+      },
+    },
+    {
+      table_id          = "bar",
+      schema            = "sample_bq_schema.json",
+      time_partitioning = null,
+      expiration_time   = 2524604400000, # 2050/01/01
+      clustering        = [],
+      labels = {
+        env      = "devops"
+        billable = "true"
+        owner    = "joedoe"
+      },
+    }
+  ]
+  external_tables = [
+    {
+      table_id              = "csv_example"
+      autodetect            = true
+      compression           = null
+      ignore_unknown_values = true
+      max_bad_records       = 0
+      source_format         = "CSV"
+      schema                = null
+      expiration_time       = 2524604400000 # 2050/01/01
+      labels = {
+        env      = "devops"
+        billable = "true"
+        owner    = "joedoe"
+      }
+      source_uris = ["gs://terraform-gcp-bigquery-external-data/bigquery-external-table-test.csv"]
```

Comment:
Please add a comment on these source URIs pointing to https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/872.